stdlib/public/Python/NumpyConversion.swift

```diff
@@ -31,50 +32,63 @@ public protocol NumpyScalarCompatible {
   /// The NumPy scalar types that this type is bitwise compatible with. Must
   /// be nonempty.
   static var numpyScalarTypes: [PythonObject] { get }
+  /// The ctypes scalar type that this type can be cast to.
+  static var ctype: PythonObject { get }
```

Comment:
This requirement is added to a public protocol. It is convenient, but isn't necessarily a good API choice because 1. the conversion is possible even without `ctypes` and 2. this requirement is not being used at all within the Python module.

Reply:
Switched over to adding makeNumpyArray() on array. I think that ctypes is necessary to keep there from being many individual calls populating the array (worst-case through python objects themselves).